src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -77,8 +80,34 @@ namespace AppInstaller::CLI::Workflow
     {
         const auto& manifest = context.Get<Execution::Data::Manifest>();
 
+        // Channel is moved to installer level, get all channels from all installers

```

Comment:
>// Channel is moved to installer level, get all channels from all installers [](start = 8, length = 76)

Channel is not an installer level item.  It is a part of the version.  If the schema has this change, I obviously missed it in review.